include/touchscreen_list_menu.h

```diff
@@ -7,6 +7,12 @@
 #include "palette.h"
 #include "touchscreen.h"
 
+enum TouchscreenListMenuTextAlignment {
+    TSMENU_ALN_LEFT,
```

Comment:
I'd prefer a longer and more descriptive value
(at least expand ALN to ALIGN)

Reply:
this is pointless bikeshedding

Reply:
grumble, fiiiiiiine